docs/airflow/production-maintenance.md

```diff
@@ -40,8 +40,7 @@ DAGs are listed in alphabetical order, as they appear in the Airflow UI.
 `gtfs_schedule_history2` | Yes | **📆 Yes** | **🔂 No** | `gtfs_ loader` | |
 `parse_and_validate_rt` | Yes | No | **🔂 No** | N/A | |
 `payments_loader` | Yes | No | Yes | N/A | |
-`payments_views` | Yes | No | Yes | `payments_ views_ staging`| |
-`payments_views_staging` | Yes | No | Yes | `payments_ loader` | |
+`payments_views` | Yes | No | Yes | | `payments_loader` |
```

Comment:
I think you can remove the `payments_loader` note here -- AFAICT there's no actual enforced dependency (the dependencies listed in this table are just meant to be actual enforced dependencies, not necessarily all "logical" data flow dependencies.)